build_deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I guess we should still have two images, though we can share the same docker. Can we get it confirmed with app-sre?

Reply:
This is not required, I confirmed with them.